x/todos/todo.rb

```diff
@@ -22,6 +22,8 @@ def initialize(data)
 
     def with_implementations
       todos.select { |x| x.implementations.any? }
+           .sort_by { |x| x.implementations.count }
```

Comment:
Can we do the reversing in-place while sorting? Maybe something like `x.implementations.count * -1`.